### DISCUSSION TITLE
No effect applied when using `min` and `max` properties on `mode="range"` in v9.0.8

### DISCUSSION THREAD

**Discussion by vildantursic** [gpbl/react-day-picker#2432] [Support]
## Code 

```tsx
import { DayPicker } from "react-day-picker";
import { sub, endOfDay, add } from "date-fns";

const DATE_RANGE_LIMIT = 4;

export function Start() {
  const [dateRange, setDateRange] = useState<DateRange>({
            from: sub(new Date(), { days: 1, hours: 1 }),
            to: endOfDay(new Date()),
        })

  return (
     <DayPicker
           mode="range"
           pagedNavigation
           numberOfMonths={2}
           showOutsideDays={false}
           defaultMonth={dateRange?.from}
           selected={dateRange}
           onDayClick={(date) => console.log(date)}
           endMonth={add(new Date(), { months: 1 })}
           disabled={[
              { after: add(new Date(), { days: 1 }) },
           ]}
           max={DATE_RANGE_LIMIT}
       />
  );
}
```

## Expected Behavior

Max should disable dates after x days from selected date. So if I select `1.1.2024.` and max is `4` last date that I should be able to select is `1.4.2024.`.

## Actual Behavior

All dates are available for selection

## Temporary solution

Add another object in `disabled` list `{ after: add(dateRange.from || new Date(), { days: DATE_RANGE_LIMIT - 1 }), before: dateRange.from }` which will disable dates after selected item.

**Comment by gpbl** [comment]
Hi @vildantursic in the previous version of DayPicker, we were indeed disabling dates when they could not belong to the selected range. This behavior caused some confusion and we removed it in v9. 

Is what you are referring to?

I think your solution of implementing a `disabled` modifiers is correct here.
